### PR TITLE
Card component with containerlist component sample

### PR DESCRIPTION
**Pull request by kotaaaa**
## Overviews of implementation

- Card component for common usecase
- Usecase example of card component at containerlist component

## Review points
- usability of card component
- Usecase implementation of card component

### CODE REVIEW

File: src/features/groups/components/ContainerList.tsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
+import { Card } from '@/components/ui';
+import { Icon } from '@/components/ui';
```
Comment (uskayyyyy) [inline src/features/groups/components/ContainerList.tsx:2]: Did you purposely separate these two lines?

File: src/features/groups/components/ContainerList.tsx [hunk 2]
```diff
@@ -9,7 +12,14 @@ export default async function ContainerList({ id }: { id: string }) {
       {/* <MenuMeatBall /> */}
       {/* <MenuPlus /> */}
       {containers.map((container) => (
-        <div key={container.id}>{container.name},</div>
+        <Card key={container.id}>
+          <div className="flex gap-4 items-center">
+            <span className="flex items-center justify-center bg-accent rounded-full w-11 h-11">
+              <Icon iconName="container" color="black" />
+            </span>
+            <span>{container.name}</span>
```
Comment (uskayyyyy) [inline src/features/groups/components/ContainerList.tsx:20]: After conducting some research, I found that both `<div>` and `<span>` tags are semantically neutral. However, when used as a child element of a flexbox, `<div>` tags are commonly used for creating layouts, while `<span>` tags are usually used to wrap text or other inline elements.

Based on this information, my suggestion is to use `<div>` to wrap the `<Icon>` tag, and use `<span>` (or no tags at all) for `{container.name}`. If you do not apply any styles or attributes to the `<span>` tag, you do not even need to enclose the content with it.

**Review by uskayyyyy — APPROVED** [review]
Thanks for implementing it! I approved it because the primary implementation of the card is excellent and versatile enough. Please take a look at the comments I left and adjust the code if you think it's needed.

**Review by uskayyyyy — CHANGES_REQUESTED** [review]
Thank you for fixing them!
Could you fix the following two files, as well? They should still include `default export`.'

- src/features/groups/components/ContainersCount.tsx
- src/features/groups/components/UserCount.tsx

File: src/features/groups/components/ContainerList.tsx (hunk 1 above)
Comment (kotaaaa) [reply src/features/groups/components/ContainerList.tsx]: Thank you for checking!! Fixed!!

File: src/features/groups/components/ContainerList.tsx (hunk 2 above)
Comment (kotaaaa) [reply src/features/groups/components/ContainerList.tsx]: Thank you for checking!! Fixed!!

**Review by uskayyyyy — APPROVED** [review]

**Pull request merged**